scripts/operations/node_management/ncn-ims-image-upload.sh

```diff
@@ -122,4 +136,14 @@ cray ims images update "$IMS_IMAGE_ID" \
         --link-etag "${MANIFEST_ETAG}" \
         --link-path "s3://boot-images/$IMS_IMAGE_ID/manifest.json" > /dev/null
 
+podman run --rm --name ncn-cpc \
+    --user root \
+    -e PRODUCT=csm \
+    -e PRODUCT_VERSION=$CSM_RELEASE \
+    -e YAML_CONTENT_STRING="{images: {$IMS_IMAGE_NAME: {id: $IMS_IMAGE_ID}}}" \
```

Comment:
I'm not sure how this would behave, but you might want to quote `$IMS_IMAGE_NAME` in this string in case it contains any weird characters. That is, maybe escape double quotes inside this string. I think that would ensure that the key is always valid and of string type. You might want to do the same for `IMS_IMAGE_ID` too.

Reply:
Makes sense.  I tried it on surtur to make sure there were no unexpected issues.